jdaviz/configs/default/plugins/model_fitting/tests/test_fitting.py

```diff
@@ -504,9 +509,10 @@ def test_cube_fit_after_unit_change(cubeviz_helper, solid_angle_unit):
     # Check that the parameter is using the current units when initialized
     assert mf._obj.component_models[0]['parameters'][0]['unit'] == f'MJy / {solid_angle_string}'
 
+    n_cpu = 1
     with warnings.catch_warnings():
         warnings.filterwarnings('ignore', message='Model is linear in parameters.*')
-        mf.calculate_fit()
+        mf.calculate_fit(n_cpu=n_cpu)
```

Comment:
```suggestion
        mf.calculate_fit(n_cpu=1)
```
Why use a separate line above to define `n_cpu` when this is the only place it's used? Same comment applies throughout.

Reply:
Out of habit I guess but you're right, it's not necessary, I'll hardcode.